eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -1,7 +1,14 @@
 param(
+  [Parameter(Mandatory = $true)]
   $RepoOwner,
+  # Use this if a pull request might have been opened from one repo against another.
+  # E.g Pull request opened from azure-sdk/azure-sdk prBranch --> Azure/azure-sdk baseBranch
+  $RepoOwnerTwo,
```

Comment:
This should really only be interesting for forks. So lets call these parameters $RepoOwner and $ForkRepoOwner.